e2e/resources/workbench-config.ts

```diff
@@ -90,7 +90,7 @@ const stable: IEnvConfig = {
 // workbench perf environment
 const perf: IEnvConfig = {
   LOGIN_URL_DOMAIN_NAME: process.env.PERF_LOGIN_URL || 'https://all-of-us-rw-perf.appspot.com',
-  API_URL: process.env.PERF_API_URL || 'https://api-dot-all-of-us-rw-perf.appspot.com/v1',
+  API_HOSTNAME: process.env.API_HOSTNAME || 'https://api-dot-all-of-us-rw-perf.appspot.com',
```

Comment:
nit: Technically if you're including the https, I would call this  `API_BASE_URL` or `API_URL`. Without the scheme - hosname